devtools.gemspec

```diff
@@ -19,17 +19,17 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'anima',        '~> 0.3.x'
   gem.add_runtime_dependency 'concord',      '~> 0.1.x'
   gem.add_runtime_dependency 'adamantium',   '~> 0.2.x'
-  gem.add_runtime_dependency 'rspec',        '~> 3.3.0'
-  gem.add_runtime_dependency 'rspec-core',   '~> 3.3.0'
+  gem.add_runtime_dependency 'rspec',        '~> 3.4.0'
+  gem.add_runtime_dependency 'rspec-core',   '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-its',    '~> 1.2.0'
   gem.add_runtime_dependency 'rake',         '~> 10.4.2'
   gem.add_runtime_dependency 'yard',         '~> 0.8.7.6'
   gem.add_runtime_dependency 'flay',         '~> 2.6.1'
   gem.add_runtime_dependency 'flog',         '~> 4.3.2'
-  gem.add_runtime_dependency 'reek',         '~> 3.5.0'
-  gem.add_runtime_dependency 'rubocop',      '~> 0.34.2'
+  gem.add_runtime_dependency 'reek',         '~> 3.6.1'
+  gem.add_runtime_dependency 'rubocop',      '~> 0.35.1'
   gem.add_runtime_dependency 'simplecov',    '~> 0.10.0'
   gem.add_runtime_dependency 'yardstick',    '~> 0.9.9'
-  gem.add_runtime_dependency 'mutant',       '~> 0.8.5'
+  gem.add_runtime_dependency 'mutant',       '~> 0.8.8'
   gem.add_runtime_dependency 'mutant-rspec', '~> 0.8.2'
```

Comment:
You update `mutant` but not mutant-rspec? A new mutant-rspec was released covering `rspec-3.4.x` also.